example/hello_object_detection_ssd_node/main.js

```diff
@@ -0,0 +1,277 @@
+  input_info.setPrecision('u8');
+  const output_info = outputs_info[0];
+  showBreakline();
+  console.log(`Start to read image from ${image_path}.`);
+  const image = await jimp.read(image_path);
+  console.log(`Succeeded.`);
+  const input_dims = input_info.getDims();
+  const input_height = input_dims[2];
+  const input_width = input_dims[3];
+  if (image.bitmap.height !== input_height &&
+      image.bitmap.width !== input_width) {
+    console.log(`Resize image from (${image.bitmap.height}, ${
+        image.bitmap.width}) to (${input_height}, ${input_width}).`);
+    image.resize(input_width, input_height, jimp.RESIZE_BILINEAR);
+  }
+  showBreakline();
+  console.log(`Check ${device_name} plugin version:`);
+  showPluginVersions(core.getVersions(device_name));
+  console.log(`Start to load network to ${device_name} plugin.`)
+  start_time = performance.now();
+  const exec_net = await core.loadNetwork(net, device_name);
+  const load_network_time = performance.now() - start_time;
+  highlight(`Succeeded: load network took ${load_network_time.toFixed(2)} ms.`);
+  showBreakline();
+  let infer_req;
+  let input_time = [];
+  let infer_time = [];
+  console.log(`Start to infer ${sync ? '' : 'a'}synchronously for ${
+      iterations} iterations.`);
+  for (let i = 0; i < iterations; i++) {
+    start_time = performance.now();
+    infer_req = exec_net.createInferRequest();
+    const input_blob = infer_req.getBlob(input_info.name());
+    const input_data = new Uint8Array(input_blob.wmap());
+    image.scan(
+        0, 0, image.bitmap.width, image.bitmap.height, function(x, y, idx) {
+          // Convert from RGBA to BGR (IE default)
+          let i = Math.floor(idx / 4) * 3;
+          input_data[i + 2] = image.bitmap.data[idx + 0];  // R
+          input_data[i + 1] = image.bitmap.data[idx + 1];  // G
+          input_data[i + 0] = image.bitmap.data[idx + 2];  // B
+        });
+    input_blob.unmap();
+    input_time.push(performance.now() - start_time);
+    start_time = performance.now();
+    if (sync) {
+      infer_req.infer();
+    } else {
+      await infer_req.startAsync();
+    }
+    infer_time.push(performance.now() - start_time);
+  }
+  const average_input_time =
+      input_time.reduce((acc, v) => acc + v, 0) / input_time.length;
+  const average_infer_time =
+      infer_time.reduce((acc, v) => acc + v, 0) / infer_time.length;
+  highlight(`Succeeded: the average inference time is ${
+      average_infer_time.toFixed(2)} ms.`);
+  highlight(`           the throughput is ${
+      (1000 / average_infer_time).toFixed(2)} FPS.`);
+  let labels = undefined;
+  try {
+    const data = await fs.readFile(labels_path, {encoding: 'utf-8'});
+    labels = data.split('\n');
+  } catch (error) {
+    warning(error);
+  }
+  const output_blob = infer_req.getBlob(output_info.name());
+  const output_data = new Float32Array(output_blob.rmap());
+  const results = topSSDResults(output_data, threshold, inputs_info[0].getDims());
```

Comment:
I think it should use `output_info.getDims()`.

Reply:
Fixed in commit: "Fixed getting output dims. Fix typos in model name"